contracts/eden/src/eden-micro-chain.cpp

```diff
@@ -157,6 +157,7 @@ struct status
    std::string memo;
    eosio::block_timestamp nextElection;
    uint16_t electionThreshold = 0;
+   uint16_t electionParticipants = 0;
```

Comment:
Please use `numElectionParticipants`. This allows us to add `electionParticipants` in the future if needed to list the participants.

Reply:
done.